CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Fixed
 - Fix issue of TypeError: cannot unpack non-iterable PDFObjRef object, when unpacking the value of 'DW2' ([#529](https://github.com/pdfminer/pdfminer.six/pull/529))
 - `PermissionError` when creating temporary filepaths on windows when running tests ([#469](https://github.com/pdfminer/pdfminer.six/issues/469))
+- Fix issue of Some Chinese characters can not be extracted correctly ([#566](https://github.com/pdfminer/pdfminer.six/issues/566))
```

Comment:
"Some Chinese" -> "some Chinese". 

Also refer to the PR number (593) instead of the issue.